pull_request_template.md

```diff
@@ -1,10 +1,12 @@
 # Description
 
```

Comment:
The PR template should not be changed. The PR puts in the desired information. This needs to be reverted back to the previous version without changes.